tensorflow/tools/dockerfiles/assembler.py

```diff
@@ -656,12 +667,23 @@ def main(argv):
               args=(FLAGS.hub_repository, dock, image, tag))
           p.start()
 
+      if not tag_failed:
+        succeeded_tags.append(tag)
+
   if failed_tags:
     eprint(
         '> Some tags failed to build or failed testing, check scrollback for '
         'errors: {}'.format(','.join(failed_tags)))
     exit(1)
 
+  if FLAGS.write_tags_to:
+    eprint('> Writing built{} tags to {}.'.format(
+        ' and tested' if FLAGS.run_tests_path else '',
+        FLAGS.write_tags_to))
+    with open(FLAGS.write_tags_to, 'w') as f:
```

Comment:
Maybe add something like...

dirname = os.dirname(LAGS.write_tags_to)
if not os.path.exists(dirname):
  os.makedirs(dirname)

...in case directory doesnt exist